extensions/enrichedScatterplots.py

```diff
@@ -35,6 +35,8 @@ def main():
     p.add_option('--plotLog', default=False, action="store_true", help="Use if you want axes to be shown on a log-scale. [False]")
     p.add_option('--negColor', default="#1b9e77", help='Color to use for Unenriched peptides. [#1b9e77]')
     p.add_option('--posColor', default="#d95f02", help='Color to use for Enenriched peptides. [#d95f02]')
+    p.add_option('--ptsSize', default="10", help='Size to use for plotted points. [10]')
+    p.add_option('--ptsTrans', default="0.5", help='Transparency to use for plotted points. [0.5]')
```

Comment:
Instead of reading these parameters in as strings and then converting when used, use the 'type' option in optparse to automatically read these in as floats. You will also need to remove the "" around the default values.